spec.bs

```diff
@@ -1419,6 +1454,60 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
   1. If |buyer| is failure, then return failure.
   1. [=map/Set=] |auctionConfig|'s [=auction config/per buyer priority signals=][|buyer|] to
     |signals|.
+1. If |config|["{{AuctionAdConfig/auctionReportBuyerKeys}}"] [=map/exists=]:
+  1. Let |interestGroupBuyers| be |auctionConfig|'s [=auction config/interest group buyers=].
+  1. If |interestGroupBuyers| is null, set |interestGroupBuyers| to a new [=list=].
+  1. [=list/For each=] |index| of [=the exclusive range|the range=] 0 to |config|[
+    "{{AuctionAdConfig/auctionReportBuyerKeys}}"]'s [=list/size=], exclusive:
+    1. Let |key| be |config|["{{AuctionAdConfig/auctionReportBuyerKeys}}"][|index|].
+    1. If |key| is not [=set/contained=] in [=the exclusive range|the range=] 0 to 2<sup>128</sup>,
+      exclusive, [=exception/throw=] a {{TypeError}}.
+    1. If |index| is equal to or greater than |interestGroupBuyers|' [=list/size=],
+      [=iteration/continue=].
+
+        Note: [=iteration/Continue=] is used (instead of [=iteration/break=]) to match validation

```

Comment:
I'm not sure if I understand this sentence correctly. Does it mean "to validate all given buyer keys"?